PHASE_1/API_SourceCode/src/api/schemas/Report.schema.ts

```diff
@@ -0,0 +1,5 @@
+import Joi from "@hapi/joi";
+
+export const ReportSchema = Joi.object({
```

Comment:
we aren't using this schema anywhere? for get requests where the only input is in params or path, we don't need a schema